mne/preprocessing/ica.py

```diff
@@ -141,48 +144,44 @@ class ICA(ContainsMixin):
         are passed to the ICA algorithm during fitting. If :class:`int`, must
         not be larger than ``max_pca_components``. If :class:`float` between 0
         and 1, the number of components with cumulative explained variance
+        (of the ``n_pca_components`` components)
         greater than ``n_components`` will be used. If ``None``,
-        ``max_pca_components`` will be used. Defaults to ``None``; the actual
+        ``n_pca_components`` will be used. Defaults to ``None``; the actual
         number used when executing the :meth:`ICA.fit` method will be stored in
         the attribute ``n_components_`` (note the trailing underscore).
-    max_pca_components : int | float | None
-        The number of principal components (from the pre-whitening PCA step)
-        that are retained for later use (i.e., for signal reconstruction in
-        `~mne.preprocessing.ICA.apply`; see the ``n_pca_components``
-        parameter). Use this parameter to reduce the dimensionality of the
-        input data via PCA before any further processing is performed.
-        If :class:`float` between 0 and 1, the number of components with
-        cumulative explained variance greater than ``max_pca_components`` will
-        be used. If ``1.0`` or ``None``, no dimensionality reduction occurs and
-        ``max_pca_components`` will equal the number of channels in the
-        `~mne.io.Raw` or `~mne.Epochs` object passed to
-        `~mne.preprocessing.ICA.fit`. Defaults to ``None``. When calling
-        `~mne.preprocessing.ICA.fit`, the attribute ``max_pca_components_``
-        will be set to the absolute number of retained PCA components.
 
         .. versionchanged:: 0.22
-            Added support for float to select components by cumulative
-            explained variance.
+           The number of components returned for a :class:`python:float`
+           is greater than the given variance level instead of less than or
+           equal to it.
```

Comment:
this sentence is quite mysterious I fear.

Maybe something like:

If :class:`python:float` it must be less than 1 and it corresponds to the threshold
of explained variance. For example if 0.8 is passed than the number of components
will explain at least 80% of the variance.